### ISSUE TITLE
Needs Some Design Sauce

### ISSUE THREAD

**Issue by greenisus** [mwilbanks/Open-Source-Smoke#1]
I'm sure your BBQ sauce is great, but the design sauce is lacking here.  You could get started with GitHub Pages and then Twitter Bootstrap!

http://pages.github.com
http://getbootstrap.com

**Comment by mwilbanks** [comment]
Agreed! Like all the BBQ here, that will be a constant work in progress too I'm sure. :)
